backend/models/user.model.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, Sequelize) => {
+    const User = sequelize.define("users", {
+        user_id: {
+            type: Sequelize.INTEGER,
+            primaryKey: true
+        },
+        name: {
+            type: Sequelize.STRING
+        },
+        password: {
+            type: Sequelize.STRING
+        },
+        salt: {
```

Comment:
Bcrypt hashing library has an inbuilt salt. So if we're using bcrypt, we might not need the salt as an attribute to users entity

Reply:
Noted, thanks for this.
Will remove from the model! 
I feel we still need to discuss the model specifics in detail during our meeting later today!